hypertrace-graphql-service/src/main/java/org/hypertrace/graphql/service/GraphQlServiceFactory.java

```diff
@@ -0,0 +1,38 @@
+package org.hypertrace.graphql.service;
+
+import java.util.List;
+import org.hypertrace.core.graphql.spi.config.GraphQlServiceConfig;
+import org.hypertrace.core.serviceframework.http.HttpContainerEnvironment;
+import org.hypertrace.core.serviceframework.http.HttpHandlerDefinition;
+import org.hypertrace.core.serviceframework.http.HttpHandlerDefinition.CorsConfig;
+import org.hypertrace.core.serviceframework.http.HttpHandlerFactory;
+import org.hypertrace.graphql.config.HypertraceGraphQlServiceConfig;
+import org.hypertrace.graphql.impl.GraphQlFactory;
+
+public class GraphQlServiceFactory implements HttpHandlerFactory {
+  private static final String SERVICE_NAME = "hypertrace-graphql";
+
+  @Override
+  public List<HttpHandlerDefinition> buildHandlers(HttpContainerEnvironment environment) {
```

Comment:
isn't this in the hypertrace-core-graphql or is it needed

Reply:
it's in both - each defines its own service (and docker image) as legacy of when hypertrace and hypertrace-core had independent platforms.